itest/assertions.go

```diff
@@ -1886,7 +1886,7 @@ func AssertGenesisOutput(t *testing.T, output *taprpc.ManagedUtxo,
 }
 
 func AssertAssetBalances(t *testing.T, client taprpc.TaprootAssetsClient,
-	simpleAssets, issuableAssets []*taprpc.Asset) {
+	simpleAssets, issuableAssets []*taprpc.Asset, includeLeased bool) {
```

Comment:
Since we should now have the same flags for the `ListAssets` and `ListBalances` calls, I wonder if we should also add a check here that makes sure the sum of all assets returned by `ListAssets` matches the sum returned in `ListBalances`?

Reply:
Your wish is my command.